aquadoggo/src/graphql/client/u64_string.rs

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use core::fmt;
+
+use serde::de::{self, Unexpected, Visitor};
+
+/// Serialise log id as strings.
+///
+/// To be used as a parameter for Serde's `with` field attribute.
+#[allow(dead_code)]
+pub mod log_id_string_serialisation {
```

Comment:
This is very cool! It's similar to what I was trying to do in the context of storing and retrieving p2panda types from the db where they are mostly strings: https://github.com/p2panda/aquadoggo/issues/89

I prefer your approach as it seems more generally useful. I don't know if it is a drop-in solution to for the db stuff though. Definitely let's keep an eye on it and see if we want to implement it in `p2panda-rs` and then try to use for the db models.